core/src/main/java/com/linecorp/armeria/common/RequestContext.java

```diff
@@ -435,6 +435,24 @@ default ByteBufAllocator alloc() {
     @MustBeClosed
     SafeCloseable push();
 
+    /**
+     * Immediately run a given {@link Runnable} with this context.
+     */
+    default void run(Runnable runnable) {
+        try (SafeCloseable ignored = push()) {
+            runnable.run();
+        }
+    }
+
+    /**
+     * Immediately call a given {@link Callable} with this context.
+     */
+    default <T> T call(Callable<T> callable) throws Exception {
```

Comment:
```suggestion
    default <T> T run(Callable<T> callable) throws Exception {
```

A runnable changes to a callable in a lambda just by the presence of return or not - I guess allowing them to change without changing this method name is more convenient than matching the type name (which is rarely used since Java 8+)

Reply:
I agree with you! Let me fix it!